### ISSUE TITLE
Actions.prevScene is current scene

### ISSUE THREAD

**Issue by HenryDavies** [aksonov/react-native-router-flux#3423]
### Version
Tell us which versions you are using:

- react-native-router-flux v4.0.6
- react v16.7.0
- react-native v0.58.3

### Expected behaviour
Actions.prevScene shows previous scene name

### Actual behaviour
Actions.prevScene shows the current scene name (same as Actions.currentScene).

### Steps to reproduce
1. Console.log(Actions.prevScene, Actions.currentScene)

**Comment by amitbravo** [comment]
My Actions.currentScene is giving me previous currentScene name .

**Comment by amitbravo** [comment]
Do you still experience this problem ?

**Comment by haorh** [comment]
I see `Actions. prevScene` value is different from  `Actions.currentScene` only during quick scene transition in `onBack`/`backAndroidHandler`, ei spamming back button.

**Comment by nabilfreeman** [comment]
Yes, I see this as well. I guess this is incomplete and therefore not documented.

**Comment by Shoikot33** [comment]
My Actions.currentScene is giving me previous scene name.
